cal/cal/models.py

```diff
@@ -583,20 +584,38 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
+    def get_hours_per_week(self, timezone, calendar_ids=None, start=None, end=None):
         """
         Returns a list of week-hour tuples corresponding to the events in this ColorCategory.
         Each week starts at the start time.
         """
         week_hours = []
         events = self.query().order_by('start')
         i = 0
-        start = events[0].start
+        # Convert start to local time
+        start = handle_time_string(str(events[0].start), timezone)
```

Comment:
Events saved in our database are already ensured to be timezone aware (see `GEvent.save`) so we shouldn't be using `handle_time_string` because that contains a lot of extra logic designed for any kind of input. Instead you can just something like `start.astimezone(pytz.timezone(timezone_str))`.